src/utils/PerfherderGraphContainer.jsx

```diff
@@ -61,13 +61,12 @@ const tip = withStyles(tipStyles)(
       ? 'lower is better'
       : 'higher is better';
     const hgURL = URL({
-      path: 'https://hg.mozilla.org/mozilla-central/pushloghtml',
-      query: { changeset: record.revision },
+      path: `https://github.com/mozilla-mobile/fenix/commit/${record.revision}`,
```

Comment:
Also, do not use string munging to make URLs:  Although the `record.revision` happens to have URL-safe characters, this is not always the case.

Reply:
So, what other alternative one can have instead of munging, concatenation ?

Reply:
Undo this change.   

Ensure your programming environment has a keyboad shortcut that will **jump to definition**, where comments may describe the function you are using.  Ensure you have a keyboard shortcut to **find all usages** so you can see how other parts of the program use a class/function